pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -396,35 +408,57 @@ func (r *rbac) createResources(ctx context.Context) error {
 	for _, ns := range namespacesToBeReconciled {
 		logger.Infow("Inject CA bundle configmap in ", "Namespace", ns.GetName())
 		if err := r.ensureCABundles(ctx, &ns); err != nil {
-			return fmt.Errorf("failed to ensure ca bundles presence in namespace %s, %w", ns.Name, err)
+			logger.Errorf("failed to ensure ca bundles presence in namespace %s: %v", ns.Name, err)
+			if err = r.AddNsToReconcileOnError(ctx, &ns); err != nil {
+				return err
```

Comment:
@jkhelil When we return from the loop on error on a specific namespace issue, we fail to reconcile further namespaces until we fix the error on the specific namespace. So IMHO, we can continue to proceed with further namespaces, even though we see error on a namespace.

Reply:
hey @jkandasa thats why we continue juste after the check. here we return from AddNsToReconcileOnError wich is removing the label on the namespace on etcd. so if something is failing there, we dont continue otherwise the namespace will be considered as reconciled without having the prerequisite resources.

Reply:
>  we return from AddNsToReconcileOnError wich is removing the label on the namespace on etcd. so if something is failing there, we dont continue

@jkhelil I meant this

Reply:
I did not get this logic, we are not adding the label, untill we are reaching the end of this function, which means everything is created.

I think what we need is more checks whether the namespace needs to be reonciled